libs/qCC_io/LASFilter.cpp

```diff
@@ -471,7 +471,8 @@ CC_FILE_ERROR LASFilter::saveToFile(ccHObject* entity, const QString& filename,
 	writerOptions.add("scale_y", lasScale.y);
 	writerOptions.add("scale_z", lasScale.z);
 
-	writerOptions.add("filename", filename.toLocal8Bit().toStdString());
+	std::string utf8Filename = filename.toUtf8().toStdString();
```

Comment:
You shouldn't need the `toUtf8()` part.

According to [the docs](https://doc.qt.io/qt-5/qstring.html#toStdString):

> Returns a std::string object with the data contained in this QString. The Unicode data is converted into 8-bit characters using the toUtf8() function.